### DISCUSSION TITLE
Exported pdf too large and not searchable

### DISCUSSION THREAD

**Discussion by MrOnijohn** [xournalpp/xournalpp#5604] [Q&A]
I use xournal++ 1.2.3 on Linux Mint 21.1 to annotate a quite large pdf-file (~300 pages). The (swedish) text in the original file is searchable and the file size is ~200MB. The exported pdf with my notations (mainly arrows and lines, on most pages) is 1,5GB and the text is not searchable. If someone could help me and explain why this is, I would be very grateful! The file size is manageable, but not being able to search for text in the output file is a huge problem for me.

**Comment by mmaluff** [comment]
This happens to me too, sometimes. Might be related to #3177.
